client/blocks/importer/wordpress/index.tsx

```diff
@@ -1,64 +1,29 @@
-import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { useSelect, useDispatch } from '@wordpress/data';
-import React, { useEffect, useState } from 'react';
-import { convertToFriendlyWebsiteName } from 'calypso/blocks/import/util';
+import React, { useEffect } from 'react';
 import { LoadingEllipsis } from 'calypso/components/loading-ellipsis';
-import { useQuery } from 'calypso/landing/stepper/hooks/use-query';
 import { ONBOARD_STORE } from 'calypso/landing/stepper/stores';
-import { addQueryArgs } from 'calypso/lib/route';
 import { useSelector } from 'calypso/state';
 import { getUrlData } from 'calypso/state/imports/url-analyzer/selectors';
 import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
-import {
-	getSiteBySlug,
-	getSite,
-	isJetpackSite,
-	hasAllSitesList,
-} from 'calypso/state/sites/selectors';
+import { getSite, isJetpackSite, hasAllSitesList } from 'calypso/state/sites/selectors';
 import { Importer, ImportJob, StepNavigator } from '../types';
-import { ContentChooser } from './content-chooser';
 import ImportContentOnly from './import-content-only';
-import ImportEverything from './import-everything';
```

Comment:
When searching for `import-everything` I see some old comments - let's trash them, they'll no longer be relevant as they're referencing non-existing code.